### PR TITLE
RAB Pragma of CappedMath Updated

### PR DESCRIPTION
**Pull request by remedcu**
Task List:

- [ ] Added the check to include equal values to return zero in subCap (Reverted)
- [ ] Better gas optimization for mulCap (Reverted)
- [x] RAB Pragma Updated

### CODE REVIEW

File: contracts/libraries/CappedMath.sol [hunk 1]
```diff
@@ -25,10 +25,10 @@ library CappedMath {
     }
 
     /**
-     * @dev Subtracts two integers, returns 0 on underflow.
+     * @dev Subtracts two integers, returns 0 on underflow or equal values.
      */
     function subCap(uint _a, uint _b) internal pure returns (uint) {
-        if (_b > _a)
+        if (_b >= _a)
```
Comment (clesaege) [inline contracts/libraries/CappedMath.sol:31]: ">" was OK as if they are equal l34 will return 0.
_a==_b is not an edge case so better treated normally. And "a>=b" is more gas intensive than "a>b" as from a compiler perspective it expends to "a>b || a==b".

File: contracts/libraries/CappedMath.sol [hunk 2]
```diff
@@ -38,13 +38,10 @@ library CappedMath {
      * @dev Multiplies two unsigned integers, returns 2^256 - 1 on overflow.
      */
     function mulCap(uint _a, uint _b) internal pure returns (uint) {
-        // Gas optimization: this is cheaper than requiring '_a' not being zero, but the
-        // benefit is lost if '_b' is also tested.
-        // See: https://github.com/OpenZeppelin/openzeppelin-solidity/pull/522
-        if (_a == 0)
+        uint c = _a * _b;
+        if (c == 0)
```
Comment (clesaege) [inline contracts/libraries/CappedMath.sol:42]: This is incorrect, we can have "c==0" while "a!=0" and "b!=0" when there is an overflow (ex: a=2^255 and b=2). In this case your code would return 0 while the correct return would be UINT_MAX.

Comment (clesaege) [reply contracts/libraries/CappedMath.sol:42]: This includes a critical vulnerability.

File: contracts/libraries/CappedMath.sol (hunk 1 above)
Comment (remedcu) [reply contracts/libraries/CappedMath.sol]: Understood.

File: contracts/libraries/CappedMath.sol (hunk 2 above)
Comment (remedcu) [reply contracts/libraries/CappedMath.sol]: Understood.

**Pull request merged**